source/_integrations/alexa.smart_home.markdown

```diff
@@ -33,11 +31,28 @@ For Home Assistant Cloud Users, documentation can be found [here](https://www.na
 
 </div>
 
+## Amazon Alexa Smart Home
+
+##### Steps to Integrate an Amazon Alexa Smart Home Skill with Home Assistant
```

Comment:
This is the only one I was unsure of exactly what you were asking. I assume you were saying "header level is off". I ended up removing this header since it's the same as the H1 for the page and restructured the headers for the entire page. Please Confirm.